src/main/java/screens/WelcomeScreen.java

```diff
@@ -0,0 +1,52 @@
+package screens;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+//import screens.Timer.TimerMainGUI;
+
+// GUI layer
+
+public class WelcomeScreen extends JFrame implements ActionListener {
+
+    /**
+     * The main welcome window of the application, acts like a home screen
+     */
+    public WelcomeScreen() {
+
+        JLabel title = new JLabel("Welcome Potato Sloths!");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JButton timer = new JButton("Start Timer");
+        JButton viewSchedule = new JButton("View Schedule");
+        JButton createTask = new JButton("View Schedule");
+
+        JPanel buttons = new JPanel();
+        buttons.add(timer);
+        buttons.add(viewSchedule);
+        buttons.add(createTask);
+
+        timer.addActionListener(this::actionPerformedTimer);
+
+        JPanel main = new JPanel();
+        main.setLayout(new BoxLayout(main, BoxLayout.Y_AXIS));
+
+        main.add(title);
+        main.add(buttons);
+        this.setContentPane(main);
+        this.pack();
+    }
+
+    /**
+     * Action Listener for the Timer Feature
+     */
+    public void actionPerformedTimer(ActionEvent evt) {
+
+        //TimerMainGUI.displayTimerScreen();
+    }
+    public void actionPerformed(ActionEvent evt) {
+        System.out.println("Click " + evt.getActionCommand());
```

Comment:
Why does this button print to the terminal instead of returning anything or updating the view? just curious.